library/src/google-map.ts

```diff
@@ -30,9 +30,18 @@ export default class GoogleMap {
   }
 
   /**
-   * Loads the google maps script
+   * Loads the Google Maps script
    */
   private loadGoogleScript(options: GoogleMapOptions): void {
+    // Check if script tag was already added
+    if (
+      document.querySelector(
```

Comment:
What about using an `id` for the script tag and getting it by id here?

Reply:
I was thinking about this too, but then there was this three headed monkey and ... you know. I'll add an id 😄

Reply:
:D Yeah that stupid monkey problem, I'll get it! 

Happily welcomed the id 👯